### DISCUSSION TITLE
Offsets in LaserGRBL

### DISCUSSION THREAD

**Discussion by Glasairman** [arkypita/LaserGRBL#2324] [Q&A] [answered]
.. importing a gcode generated file from Inkscape.file from Inkscape.
How to shift the cut image location  in LaserGRBL  to where there is fresh material without going back to Inkscape / Gcode Extension and offsetting it there every time?
See screenshot. I found a reference to this but cannot see anywhere in LG where there is a menu item for it
![Screenshot 2024-07-24 141619](https://github.com/user-attachments/assets/1d84c7a8-c1d0-438d-899e-eb6d3ba16bb1)

**Comment by Tribo-2dot0** [comment]
o Offset é para imagens raster (png/jpg), svg já vem com todas as informações embutidas, inclusive posicionamento...
mova a cabeça do laser para uma posição desejada e defina ali seu novo Ponto Zero (ícone do globo).

**Reply by Glasairman** [reply]
It is a vector file from Inkscape, not an JPEG/PNG image

**Reply by Tribo-2dot0** [reply, marked as answer]
Sim, eu havia lido... por isso a janela Target Image é diferente...
Como eu disse, para arquivos svg (que não aparece a opção de Offset), simplesmente mova a cabeça do laser para uma posição desejada e defina ali seu novo Ponto Zero (ícone do globo).

**Reply by Glasairman** [reply]
Thanks got it now
